internal/services/compute/virtual_machine.go

```diff
@@ -425,6 +425,30 @@ func flattenVirtualMachineOSDisk(ctx context.Context, disksClient *disks.DisksCl
 	}, nil
 }
 
+func virtualMachineOsImageNotificationSchema() *pluginsdk.Schema {
+	return &pluginsdk.Schema{
+		Type:     pluginsdk.TypeList,
+		Optional: true,
+		MaxItems: 1,
+		Elem: &pluginsdk.Resource{
+			Schema: map[string]*pluginsdk.Schema{
+				"enabled": {
```

Comment:
instead of having enabled here could we just use the presence of the block to determine if it is enabled?

Reply:
Make sense. Removed `enabled`. Thanks.